configure.ac

```diff
@@ -37,7 +37,6 @@ AC_ARG_VAR(SYMBOLSUFFIX, [suffix to add to ARPACK, BLAS and LAPACK function name
 
 if test x"$SYMBOLSUFFIX" != x""; then
   dnl Need to rely on non-F77 features
-  AC_FC_LINE_LENGTH(unlimited)
```

Comment:
I think this is here for a reason. I don't remember why but we should probably just disable this line and the next when we detected we are on cygwin instead of removing it for everybody